site/docs/plugins/console-time.md

```diff
@@ -12,14 +12,19 @@ This library is intended to help with individual updates.
 ## Debug Your Implementation
 
 ```ts
-import {generateUpdateMiddleware} from 'telegraf-middleware-console-time';
+import { generateUpdateMiddleware } from "telegraf-middleware-console-time";
 
-if (process.env['NODE_ENV'] !== 'production') {
-    bot.use(generateUpdateMiddleware());
+if (process.env.NODE_ENV !== "production") {
+  bot.use(generateUpdateMiddleware());
 }
 
 // Your implementation
-bot.command('start', …);
+bot.command(
+  "start",
+  () => {
+    /* ... */
+  },
+);
```

Comment:
It looks too different to how we previously handled this.
```suggestion
bot.command('start', /* ... */);
```

Reply:
There's a formatting problem when we do `('x', /* ... */)`.

Reply:
formatting problem

Reply:
Someone suggested once to put the comma here:
```suggestion
bot.command("start" /* , ... */);
```
You decide.

Reply:
Someone accepted that suggestion!